aries-site/src/pages/learn/the-box-model-part-one.mdx

```diff
@@ -0,0 +1,227 @@
+</Box>
+```
+
+Now, the image and the Box containing the heading, text, and button are the only two direct children of the container Box, so only those elements are displayed in a row. For any content that we wanted to display in a column, we had to group it in its own Box.
+
+At this point, your project should look like this:
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/step-2.png" />
+</Box>
+
+## Use Box to manage an image's width
+
+Right now, the dimensions of the image file are determining how wide it is. In order to have more control of the image’s width, wrap the image in its own Box.
+
+```
+...
+<Box>
+  <Image src="https://www.insight.com/content/insight-web/en_US/shop/partner/hewlett-packard-enterprise/product-portfolio/servers/jcr%3Acontent/top-inner-full-width/column_layout/-column-1/image_0.img.jpg/1566699651505.jpg" />
+</Box>
+...
+```
+
+## Adjust width to manage content dimensions
+
+In order to ensure the text content remains a more readable width across wide screens, apply `width="large"` on the Box containing the heading, text, and button. To keep the button from stretching the full width of the box, apply `alignSelf="start"` on the Button. Notice how the button is no longer spanning the entire width of the Box.
+
+```
+<Box direction="row">
+  <Box width="large">
+    <Heading margin="none">
+      New HPE ProLiant Servers Engineered for your Hybrid World
+    </Heading>
+    <Text size="xlarge">
+      With a cloud operating experience, built-in security, and optimized
+      performance for your workloads to drive your business forward.
+    </Text>
+    <Button alignSelf="start" label="Learn more" kind="cta-primary" />
+  </Box>
+  <Box>
+    <Image src="https://www.insight.com/content/insight-web/en_US/shop/partner/hewlett-packard-enterprise/product-portfolio/servers/jcr%3Acontent/top-inner-full-width/column_layout/-column-1/image_0.img.jpg/1566699651505.jpg" />
+  </Box>
+</Box>
+```
+
+Note: Throughout this tutorial, we’ll be using t-shirt sizes. Make sure to read the [explanation on t-shirt sizes](/foundation/tshirt-sizing) for more information on what they are and their value in Grommet.
+
+At this point, your project should look something like this:
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/step-3.png" />
+</Box>
+
+## Add a background color and pad
+
+Now, let’s add a background color to the hero section by adding `background="background-contrast"` to the outer Box. We also want some space added between the edge of the outer Box and the content. To do so, add `pad="large"` to the outer Box.
+
+Remember that `pad` (equivalent to CSS padding) applies space _inside_ of a given container while `margin` applies space **\*\*\***outside**\*\*\*** of a given container.
```

Comment:
what were you trying to highlight here? `**\*\*\***outside**\*\*\*** `
should it just be _outside_ or were you trying to bold?

Reply:
Thanks for catching, seems like markdown mistranslated.